src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -0,0 +1,228 @@
+  })
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value)
+
+    //TODO: Add throttle
+    relay.refetch(
+      {
+        hasTerm: true,
+        term: event.target.value,
+      },
+      null,
+      error => {
+        if (error) {
+          logger.error(error)
+          return
+        }
+      }
+    )
+  }
+
+  return (
+    <AutocompleteInput
+      placeholder={isXs ? t`navbar.searchArtsy` : t`navbar.searchBy`}
+      spellCheck={false}
+      options={value.length < 2 ? [] : formattedOptions}
+      value={value}
+      onChange={handleChange}
+      onClear={() => setValue("")}
+      header={<NewSearchInputPillsFragmentContainer viewer={viewer} />}
+      // renderOption={option => <NewSearchBarSuggestion option={option} />}
+      renderOption={option => (
+        <SuggestionItem
+          display={option.text}
+          href={option.href}
+          isHighlighted={false} // TODO: change
+          label={option.subtitle}
+          imageUrl={option.imageUrl}
+          query={value}
+          showArtworksButton={option.showArtworksButton}
+          showAuctionResultsButton={option.showAuctionResultsButton}
+        />
+      )}
+      footer={
+        <NewSearchBarFooter
+          display={value}
+          href={`/search?term=${encodeURIComponent(value)}`}
+          isHighlighted={false}
+          label={value}
+          query={value}
+        />
+      }
+    />
+  )
+}
+
+export const NewSearchBarInputRefetchContainer = createRefetchContainer(
+  withSystemContext(NewSearchBarInput),
+  {
+    viewer: graphql`
+      fragment NewSearchBarInput_viewer on Viewer
+        @argumentDefinitions(
+          term: { type: "String!", defaultValue: "" }
+          hasTerm: { type: "Boolean!", defaultValue: false }
+        ) {
+        searchConnection(query: $term, mode: AUTOSUGGEST, first: 7)
+          @include(if: $hasTerm) {
+          edges {
+            node {
+              displayLabel
+              href
+              imageUrl
+              __typename
+              ... on SearchableItem {
+                displayType
+                slug
+              }
+              ... on Artist {
+                statuses {
+                  artworks
+                  auctionLots
+                }
+              }
+            }
+          }
+        }
+        ...NewSearchInputPills_viewer @arguments(term: $term)
+      }
+    `,
+  },
+  graphql`
+    query NewSearchBarInputRefetchQuery($term: String!, $hasTerm: Boolean!) {
+      viewer {
+        ...NewSearchBarInput_viewer @arguments(term: $term, hasTerm: $hasTerm)
+      }
+    }
+  `
+)
+
+interface NewSearchBarInputQueryRendererProps extends BoxProps {
+  isXs: boolean
+}
+
+export const NewSearchBarInputQueryRenderer: FC<NewSearchBarInputQueryRendererProps> = props => {
+  const { relayEnvironment, searchQuery = "" } = useContext(SystemContext)
+  const isMounted = useDidMount(typeof window !== "undefined")
```

Comment:
I think it will be enough just to check `typeof window !== "undefined"` (or even better to use [isServer](https://github.com/artsy/force/blob/25bfc0789dc42bad14afcaea14c6e8cec80a968a/src/Utils/device.ts#L1)), since with client-side rendering the condition will always be `true` and there is no need to use `useDidMount`. @damassi or am I wrong?
 
Something like this
```javascript
export const NewSearchBarInputQueryRenderer = props => {
  ...

  if (isServer) {
    return <StaticSearchContainer searchQuery={searchQuery} {...props} />
  }

  return (...)
}
```